### ISSUE TITLE
I have a data set with just 11% of positive target. Therefore, I would like to create my binary classification model after doing an undersampling. Is there a widget for this? I saw the data sampler, but looks like it is only for training and validation

### ISSUE THREAD

**Issue by saprissa2018** [biolab/orange3#3198]

**Comment by ajdapretnar** [comment]
You are right, Data Sampler doesn't include over/undersampling. It would be a feature request.

**Comment by astaric** [comment]
You can always split your data on class (using Select Rows) over/under sample one class with DataSampler and concatenate the result.

![screen shot 2018-09-12 at 11 13 45](https://user-images.githubusercontent.com/588601/45415021-f9538380-b67c-11e8-9b56-e12ee05a1769.png)

**Comment by markotoplak** [comment]
@astaric, I like your approach. Should we put it into a Data Sampler documentation?

**Comment by ajdapretnar** [comment]
I will do it.

**Comment by astaric** [comment]
If #3244 helps, you can use it, otherwise close it once you make your PR.
